internal/config/config.go

```diff
@@ -29,6 +29,10 @@ type AppConfig struct {
 	// DisableMetadata allows you to turn off collection of device
 	// metadata including last handshake time & rx/tx bytes
 	DisableMetadata bool `yaml:"disableMetadata"`
+	// When you set up a device, the configuration file that
+	// you can download has the default filename 'WireGuard.conf'
+	// We can change this name. (Do not include the '.conf' extension)
```

Comment:
I'd change this description a bit to better match the format and wording of the other ones
```suggestion
	// The name of the WireGuard configuration file that can
	// be downloaded through the web UI after adding a device.
	// Do not include the '.conf' extension
	// Defaults to 'WireGuard' (resulting full name 'WireGuard.conf')
```